kernel/src/transaction/mod.rs

```diff
@@ -718,97 +1004,204 @@ impl Transaction {
-                    data_type => Owned(StructField {
-                        name: field.name.clone(),
-                        data_type: data_type.into_owned(),
-                        nullable: true,
-                        metadata: field.metadata.clone(),
-                    }),
-                };
-                Some(field)
-            }
-        }
-
         let input_schema = scan_row_schema();
         let target_schema = NullableStatsTransform
             .transform_struct(get_log_remove_schema())
             .ok_or_else(|| Error::generic("Failed to transform remove schema"))?
             .into_owned();
         let evaluation_handler = engine.evaluation_handler();
 
-        Ok(self
-            .remove_files_metadata
-            .iter()
-            .map(move |file_metadata_batch| {
-                let transform = Expression::transform(
-                    Transform::new_top_level()
-                        .with_inserted_field(
-                            Some("path"),
-                            Expression::literal(self.commit_timestamp).into(),
-                        )
-                        .with_inserted_field(
-                            Some("path"),
-                            Expression::literal(self.data_change).into(),
-                        )
-                        .with_inserted_field(
-                            // extended_file_metadata
-                            Some("path"),
-                            Expression::literal(true).into(),
-                        )
-                        .with_inserted_field(
-                            Some("path"),
-                            Expression::column([FILE_CONSTANT_VALUES_NAME, "partitionValues"])
-                                .into(),
-                        )
-                        // tags
-                        .with_inserted_field(
-                            Some("stats"),
-                            Expression::column([FILE_CONSTANT_VALUES_NAME, TAGS_NAME]).into(),
-                        )
-                        .with_inserted_field(
-                            Some("deletionVector"),
-                            Expression::column([FILE_CONSTANT_VALUES_NAME, BASE_ROW_ID_NAME])
-                                .into(),
-                        )
-                        .with_inserted_field(
-                            Some("deletionVector"),
-                            Expression::column([
-                                FILE_CONSTANT_VALUES_NAME,
-                                DEFAULT_ROW_COMMIT_VERSION_NAME,
-                            ])
-                            .into(),
-                        )
-                        .with_dropped_field(FILE_CONSTANT_VALUES_NAME)
-                        .with_dropped_field("modificationTime"),
-                );
-                let expr = Expression::struct_from([transform]);
+        Ok(remove_files_metadata.map(
```

Comment:
Note to reviewer, this is a formatting change because of not reference self.remove_file_metadata)